sjb/actions/script.py

```diff
@@ -30,7 +30,7 @@ class ScriptAction(Action):
     the repository as the working directory.
     """
 
-    def __init__(self, repository, script, title, timeout=3600):
+    def __init__(self, repository, script, title, timeout=7200):
```

Comment:
yeah this was intended to be 6 hours but i failed at math.  i'd probably raise it to 21600 if we want to be truly conservative.  but if we're confident 7200 is enough, sure.